examples/05-plotting/00-plot_on_path.py

```diff
@@ -0,0 +1,62 @@
+"""
+.. _plot_on_path:
+
+Plot results on a specific path
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+This example shows how to get a result mapped over a specific path,
+and how to plot it.
+
+"""
+
+from ansys.dpf import core as dpf
+from ansys.dpf.core import examples
+
+###############################################################################
+# Path plotting
+# ~~~~~~~~~~~~~
+# We will use a DpfPlotter to plot a mapped result over a defined path of
```

Comment:
Consider using the `:class:` directive with `DpfPlotter`. Links in documentation are great.